src/routes/(console)/project-[project]/auth/security/updateMockNumbers.svelte

```diff
@@ -27,8 +27,7 @@
     $: isSubmitDisabled = JSON.stringify(numbers) === JSON.stringify(initialNumbers);
 
     // temporarily enable the component for design review
-    let isComponentDisabled: boolean =
-        (false && isSelfHosted) || (isCloud && $organization?.billingPlan === BillingPlan.FREE);
+    let isComponentDisabled: boolean = false && isSelfHosted || (isCloud && $organization?.billingPlan === BillingPlan.FREE);
```

Comment:
Won't the `false && isSelfHosted` will make the condition always false?

Reply:
Yea I kept it false so we can test it during QA. I will change it back before we are ready for release